src/renderer.cpp

```diff
@@ -826,7 +1088,7 @@ void Renderer::setScissor(uint16_t x, uint16_t y, uint16_t width, uint16_t heigh
     scissorHeight = getInternalCoordY(height);
 }
 
-void Renderer::setClearFlags(bool doClearColor, bool doClearDepth)
+void Renderer::setClearFlags(bgfx::ViewId viewId, bool doClearColor, bool doClearDepth)

```

Comment:
Ideally there's no need to know which view to clear. This class was created with a complexity abstraction in mind, introducing this parameter it means killing this logic. Personally I'd prefer finding another way and keep the API call as it was meant to be.

Reply:
Please see comment above. If you decide its best to revert to using `backendViewId` then I will revert this API to its previous state.

Reply:
As explained in the previous comment, I reverted to using the `backendViewId` variable so this API was also reverted.